src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    public static final String CRYPTO_ALGORITHM = "Sha-512";

```

Comment:
```suggestion
    private static final String CRYPTO_ALGORITHM = "Sha-512";
```
Follow incapsulation principal